src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,55 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+import { Movie } from '../../types/movie';
+
+type Props = {
+  movie: Movie,
+  key: string,
+};
+
+export const MovieCard: React.FC<Props> = ({ movie, key }) => {
+  const {
+    title,
+    description,
+    imgUrl,
+    imdbUrl,
+  } = movie;
+
+  return (
+    <div
+      className="card"
+      data-cy="card"
+      key={key}
```

Comment:
No, you don't need this.

Reply:
<img width="687" alt="image" src="https://user-images.githubusercontent.com/32794096/182338620-a2290ab1-d13e-4af8-b003-33a2a02019da.png">